CHANGELOG.md

```diff
@@ -2,6 +2,11 @@
 
 ## [Unreleased]
 
+### Changed
+
+- Vsock host-initiated connections now implement a trivial handshake protocol.
+  See the [vsock doc](docs/vsock.md#host-initiated-connections) for details.
```

Comment:
nit: maybe reference the github issues fixed here.

Reply:
Done.